src/routes/my_purchased_albums.jsx

```diff
@@ -0,0 +1,52 @@
+import { useRef, useState } from 'react';
+import { useLoaderData, useRevalidator } from 'react-router-dom';
+import { Album } from '~/components/album';
+import Navbar from '~/components/navbar';
+import { supabase } from '~/global';
+import { fetchSignedAlbumUrls } from '~/loaders/fetchSignedAlbumUrls';
+import { createAlbum } from '~/queries/albums';
+
+export { default as ErrorBoundary } from 'Routes/error';
+
+export const loader = async () => {
+    const { data: my_profile, error: profileError } = await supabase.from('my_profile').select().maybeSingle();
+    if (profileError) {
+        throw profileError;
+    }
+    if (my_profile.user_type !== 'Creator') {
```

Comment:
non Creators should also be able to view their purchased albums
remove this check